fclient/request.go

```diff
@@ -332,8 +332,8 @@ func ParseAuthorization(header string) (scheme string, origin, destination spec.
 		if len(pair) != 2 {
 			continue
 		}
-		name := pair[0]
-		value := strings.Trim(pair[1], "\"")
+		name := strings.TrimSpace(pair[0])
+		value := strings.TrimSpace(strings.Trim(strings.TrimSpace(pair[1]), "\""))
```

Comment:
Why does this call `TrimSpace` twice? Is once not enough in this case?

Reply:
Since we're splitting the line on `,` and `=`, we may end up with something like `destination	=  "bar"  `. This means the inner `TrimSpace` results in `"bar"`, then we `Trim` the `"`, resulting in `bar` - and while writing this, yes, one `TrimSpace` is enough.